### DISCUSSION TITLE
SerialBattery with JK-BMS Alarm High voltage

### DISCUSSION THREAD

**Discussion by OFreddy** [Louisvdw/dbus-serialbattery#674] [Q&A] [answered]
I am operating an ESS System with:
- Victron MultiPlus-II
- Raspberry with Venus OS V2.94
- Serial Battery V0.14.3
- 16 S LiFePo4 230Ah Battery
- JK-BMS

The MultiPlus-II is set to 56V Absorption Voltage and 54.4V Float Voltage.
Unfortunately the SerialBattery/JKBMS generates  a "High Voltage Alarm" at arround 54.6V causing the MP-II interupting the balancing stage.

![2023-05-26 19_27_47-Remote Console on LAN – Mozilla Firefox](https://github.com/Louisvdw/dbus-serialbattery/assets/35624593/84985e27-4e37-430b-84ea-4f1922023b7b)

Unfortunately I cannot find a setting in utils.py which seems to match to this alarm.
The OVP Voltage in the JKBMS is set to 3.6V => 57.6V.

The voltage settings in utils.py are:
MIN_CELL_VOLTAGE = 2.7
MAX_CELL_VOLTAGE = 3.5
FLOAT_CELL_VOLTAGE = 3.4

I found several issues in different groups which seem to address the same problem but without any conclusion.

Can anyone give me a hint where I can find the right setting which is causing this alarm?

**Comment by mr-manuel** [comment]
Which driver version are you using?

**Reply by OFreddy** [reply]
Thanks for the reply.
I am using the serialbattery driver version V0.14.3 which was installed when selecting option 1 in die install script.

**Reply by mr-manuel** [reply]
Could you try option 3 and then 2?

What are your cell voltages? Could you post a screenshot of the cell voltages page?

**Reply by OFreddy** [reply]
Now the alarm triggered again.
![2023-05-27 13_19_31-Remote Console on LAN – Mozilla Firefox](https://github.com/Louisvdw/dbus-serialbattery/assets/35624593/e3b0b239-f76d-4fee-94f8-4e24d8c7ab38)
Is it because the cell voltage of single cells possibly exceeded 3.6 V?
![2023-05-27 13_17_43-Remote Console on LAN – Mozilla Firefox](https://github.com/Louisvdw/dbus-serialbattery/assets/35624593/31aace84-7f74-47b5-9807-c01581d63b25)

**Reply by OFreddy** [reply, marked as answer]
The take a look at the single cell voltages was the hint I needed.
The exceeded single cell voltage was the reason for the alarm message.
I was confused by the message text indicating the overall voltage. I was not able to locate such a limit anywhere.

Thanks for the fast and helpfull reply. I am very happy that the problem is solved.

**Reply by mr-manuel** [reply]
Unfortunately there is no high cell voltage error message in Venus OS. We could trigger also a cell imbalance warning
